src/main/java/com/vmtapp/enterprise/dao/PhotoRepository.java

```diff
@@ -3,5 +3,11 @@
 import com.vmtapp.enterprise.dto.Photo;
 import org.springframework.data.repository.CrudRepository;
 
+import java.util.Optional;
+
+
 public interface PhotoRepository extends CrudRepository<Photo, Integer> {
+
+    @Override
+    Optional<Photo> findById(Integer integer);
```

Comment:
Add a variable in the CrudRepository. Source:https://martinfowler.com/bliki/TechnicalDebt.html